tests/functional/functional.yaml

```diff
@@ -185,6 +185,8 @@ tester-config:
   - SIGTERM_ALL
   - SIGQUIT_AND_REMOVE_ONE_FOLLOWER
   - SIGQUIT_AND_REMOVE_ONE_FOLLOWER_UNTIL_TRIGGER_SNAPSHOT
+  - SIGKILL_FOLLOWER
```

Comment:
`SIGKILL_ONE_FOLLOWER` so as to keep the naming consistent as the existing `SIGTERM_ONE_FOLLOWER` and `SIGQUIT_AND_REMOVE_ONE_FOLLOWER` ?